src/api/hooks/useGetANS.ts

```diff
@@ -22,6 +22,12 @@ export function useGetNameFromAddress(address: string) {
   const [name, setName] = useState<string | undefined>();
 
   useEffect(() => {
+    const cachedName = localStorage.getItem(address)
```

Comment:
awesome lgtm! could you run `yarn fmt -w` to make sure format looks good?

Reply:
husky looks pretty solid to me. we will rewrite but mostly its gonna be to migrate to the nextjs framework. is husky compatible with pnpm thou? only npm & yarn are mentioned on the site.

Reply:
looks husky [supports](https://typicode.github.io/husky/#/?id=automatic-recommended) pnpm, we can always try at least, but if migrating to nextjs will result in a new repo, then ill hold.

Reply:
nextjs migration will be done by third party so not sure at this point. lets try husky then